scripts/write_deployinst_input.py

```diff
@@ -270,6 +310,7 @@ def input_render(init_date, duration, reactor_file,
     A complete cylus input file.
 
     """
+    template = read_template('../templates/input_template.xml.in')
```

Comment:
why isn't this filename a parameter anymore? hardcoding it is not very flexible.